### PR TITLE
[fix](Nereids) fix plan shape of query64 unstable 

### PR DESCRIPTION
**Pull request by keanji-x**
## Proposed changes

don't remove the physical plan after optimizing the plan in dphyper

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by XieJiann** [comment]
run p0

**Comment by XieJiann** [comment]
run buildall

**Comment by XieJiann** [comment]
run buildall

**Comment by XieJiann** [comment]
run buildall

**Comment by XieJiann** [comment]
run buildall

**Comment by XieJiann** [comment]
run buildall

**Comment by XieJiann** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Review by jackwener — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ class CostModelV1 extends PlanVisitor<Cost, PlanContext> {
     // the penalty factor is no more than BROADCAST_JOIN_SKEW_PENALTY_LIMIT
     static final double BROADCAST_JOIN_SKEW_RATIO = 30.0;
     static final double BROADCAST_JOIN_SKEW_PENALTY_LIMIT = 2.0;
-    private int beNumber = 1;
+    private final int beNumber;
```
Comment (englefly) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java:71]: 这个是测试用的变量，来自sessionVariable，为什么要final

Comment (XieJiann) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java:71]: 这个是在costModel初始化的时候设定的，中间不会被修改

**Pull request merged**